config/environments/development.rb

```diff
@@ -126,6 +127,7 @@
     Bullet.add_whitelist(type: :unused_eager_loading, class_name: "ApiSecret", association: :user)
     # acts-as-taggable-on has super weird eager loading problems: <https://github.com/mbleigh/acts-as-taggable-on/issues/91>
     Bullet.add_whitelist(type: :n_plus_one_query, class_name: "ActsAsTaggableOn::Tagging", association: :tag)
+    Bullet.add_whitelist(type: :unused_eager_loading, class_name: "Article", association: :top_comments)
```

Comment:
Whitelisted the association here as Bullet was flip flopping between telling me to include and not included. Seems a riff on this problem: https://github.com/flyerhzm/bullet/issues/147